addons/hr_attendance/static/src/client_action/kiosk_confirm/kiosk_confirm.xml

```diff
@@ -1,5 +1,3 @@
-
```

Comment:
this is unnecessary change, especially in stable , to be removed

Reply:
thanks @sofiagvaladze ,
updated the pr